content/docs/04-clusters/01-public-cloud/01-aws.md

```diff
@@ -946,6 +963,33 @@ To create an AWS cloud account using STS credentials follow the steps below:
 </Tabs.TabPane>
 </Tabs>
 
+## Create AWS US Gov Cloud Account
+
+ To deploy the AWS US Gov Clusters in the Palette Platform, having the AWS US Gov Cloud Account registered to the Palette Platform is a significant prerequisite. This section guides you on creating an AWS US Gov cloud account in the Palette console.

```

Comment:
```suggestion
Palette supports integration with [AWS GovCloud (US)](https://aws.amazon.com/govcloud-us/?whats-new-ess.sort-by=item.additionalFields.postDateTime&whats-new-ess.sort-order=desc). Using Palette you can deploy Kubernetes clusters to your AWS GovCloud account. To get started with AWS GovClud and Palette, use the following steps.
```
The current paragraph form uses awkward language and is confusing to read. Keep things simple.